bitnami/kuberay/crds/ray.io_rayclusters.yaml

```diff
@@ -0,0 +1,13370 @@
+
```

Comment:
I personally like adding a comment at the top of CRDs files with the source, e.g.:
```
# https://github.com/ray-project/kuberay/blob/v1.0.0/ray-operator/config/crd/bases/ray.io_rayclusters.yaml
```

This is especially helpful to detect outdated CRDs and quickly update them. WDYT?

Reply:
Done